test_is_prime.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from is_prime import is_prime
+
+'''
+Function that tests the is_prime function in is_prime.py
+'''
+
+class TryTesting(unittest.TestCase):
+    def test_functional_1(self):
+        self.assertTrue(is_prime(11))
+    def test_functional_2(self):
+        self.assertFalse(is_prime(4))
+    def test_functional_3(self):
+        self.assertTrue(is_prime(53))
```

Comment:
i would recommend to group multiple functional tests into one test method, since otherwise the naming gets a bit "confusing" (i.e. "test_functional_1", "test_functional_2" etc.)